lua/nvcommunity/lsp/dim/init.lua

```diff
@@ -0,0 +1,12 @@
+---@type NvPluginSpec
+local spec = {
+  {
+    "narutoxy/dim.lua",
```

Comment:
Neodim or the other dim.lua better though no?

Reply:
didnt realized the repo changed the name and the neodim never worked for me, I will test again